docs/input_output.md

```diff
@@ -2,85 +2,279 @@
 
 ## General Structure
 
-GIBS-in-the-Cloud uses a common format for all inputs and outputs from Tasks consisting of a JSON object which holds all necessary information about the task execution and AWS environment. Tasks return objects identical in format to their input with the exception of a task-specific `"payload"` field. Tasks may also augment their execution metadata.
+Cumulus uses a common format for all inputs and outputs from workflow steps consisting of a JSON object which holds all necessary information about the task execution and AWS environment. Tasks return objects identical in format to their input with the exception of a task-specific `"payload"` field. Tasks may also augment their execution metadata.
```

Comment:
I _think_ that the input to the workflow itself is of the same format - it may be worth mentioning this, suggested update:
> Cumulus uses a common format for all inputs and outputs to workflows. The same format is used for input and output from workflow steps. The common format consists of a JSON object which holds...